types/passport-totp/strategy.d.ts

```diff
@@ -0,0 +1,25 @@
+export = Strategy;
+
+type DoneCallback = (...args: any[]) => void;
+
+type SetupCallback = (user: any, done: DoneCallback) => void;
+
+interface StrategyOptions {
+    codeField: string;
+    window: number;
```

Comment:
It looks like these are [both optional](https://github.com/jaredhanson/passport-totp/blob/master/lib/strategy.js#L23-L24_. `codeField` defaults to `"code"` and `window` defaults to `6`.

```suggestion
    codeField?: string;
    window?: number;
```

Reply:
Oh, missed this one, thanks!